tasks/documentation.ts

```diff
@@ -1,23 +1,48 @@
 import gulp from "gulp";
 import DocsEngine from "carbonldp-ts-docs-engine";
 import path from "path";
-import del = require('del');
+import del = require("del");
+import fs from "fs";
 
 const projectRootDir:string = path.resolve( __dirname, "../" );
 const sourceDir:string = path.resolve( projectRootDir, "src/" );
 const outputDir:string = path.resolve( projectRootDir, "api-docs/" );
+const descriptionTemplateRoute:string = path.join(outputDir, "/templates/documentation-description.njk");
```

Comment:
The template should be moved from the output directory since this is the one that is been made public as the API page, and is not good that contains the src code.

A better place would be `build/docs/templates`.